src/main/java/edu/unc/lib/boxc/migration/cdm/services/SourceFileService.java

```diff
@@ -58,11 +58,11 @@
  * @author bbpennel
  */
 public class SourceFileService {
-    private static final Logger log = getLogger(SourceFileService.class);
+    protected static final Logger log = getLogger(SourceFileService.class);
```

Comment:
I don't think we want to inherit the logger in children class, we should be able to just declare a new one in BoxctronFileService. If they shared a logger than log statements from BoxctronFileService would get logged as coming from "SourceFileService" rather than itself.